wiki/Contests/Contest_points/en.md

```diff
@@ -0,0 +1,59 @@
+# Contest points
+
+Contest points are points that are used as a reward in official contests. They are mainly used to determine the eligibility of a user to receive the *Elite Mapper* title, which is awarded to users who have reached 6 points.
```

Comment:
Probably should specify mapping contests as its only for a mapper title and only been used in mapping contests. Probably also applied to the history section further down.